docs/host_for_multiuser/setup.md

```diff
@@ -167,7 +166,7 @@ alembic upgrade head
 ```
 
 ###### &#x248a;Insert initial data
-- Same as [Insert initial data](#insert-initial-data) procedure.
+- For data input procedure, see :ref:`set-optinist-config`.
```

Comment:
remind this one.

Reply:
Should i link to Insert Initial Data in the same host_for_multiuser/setup.md file?

For the user with the Non-Docker Platforms, they still need to add an initial sql entry, right? They should do this in another terminal window, within the same conda env, right? 
They would then use 
`mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}`
instead of `docker exec -it {DB_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}`

Reply:
- It may be difficult to understand, but the following is described in the documentation, assuming the use of “mysql with docker compose” regardless of Docker/Native.
  > Below are the instructions for using mysql with docker compose.

- Also, when I build the current setup.md, it is labeled as in the image below, is this the expected format? (Broken link, no?)
  - ![image](https://github.com/user-attachments/assets/1667c6f4-0b27-4930-8d62-eeb4cad98144)

Reply:
To be honest, I think the "For Non-Docker Platforms" is not easy to understand and the links to the sections are probably incorrect. However, It is difficult for me to fix this as I do not understand what this means
> assuming the use of “mysql with docker compose” regardless of Docker/Native.

This is a different section to the update of the documentation I was doing, and I do not really understand what this section means. Maybe it would be easier if you try to fix this and make a new PR onto this branch?

Reply:
Let's match perceptions through conversation.

Reply:
I think it would be easier to add the information again, rather than sending to a different place with a link.

Reply:
1. Edit studio/config/.env

- Change SECRET_KEY to any random string.
- Change USE_FIREBASE_TOKEN to True.
- Change IS_STANDALONE to False

2. Insert initial data
In a new console window create a mysql database
`mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}`
Insert initial data:
```
INSERT INTO organization (name) VALUES ('{YOUR_ORGANIZATION_NAME}');
INSERT INTO roles (id, role) VALUES (1, 'admin'), (20, 'operator');
INSERT INTO users (uid, organization_id, name, email, active) VALUES ('{FIREBASE_USER_UID}', 1, '{YOUR_NAME}', '{YOUR_EMAIL}', true);
INSERT INTO user_roles (user_id, role_id) VALUES (1, 1);
```
Check [notes](https://github.com/arayabrain/barebone-studio/blob/develop-main/docs/host_for_multiuser/setup.md#insert-initial-data) for more information.